php_fpm/metadata.csv

```diff
@@ -1,5 +1,8 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
 php_fpm.listen_queue.size,gauge,,,,Size of the socket queue of pending connections,0,php_fpm,listen queue size
-php_fpm.processes.active, gauge,,process,, Total number of active processes,0,php_fpm, active processes
-php_fpm.processes.idle, gauge,,process,,Total number of idle processes,0,php_fpm, idle processes
-php_fpm.processes.total, gauge,,process,, Total number of processes,0,php_fpm, total processes
\ No newline at end of file
+php_fpm.processes.active,gauge,,process,, Total number of active processes,0,php_fpm,active processes
+php_fpm.processes.idle,gauge,,process,,Total number of idle processes,0,php_fpm,idle processes
+php_fpm.processes.total,gauge,,process,, Total number of processes,0,php_fpm,total processes
+php_fpm.requests.accepted,count,,request,,Total number of accepted requests,0,php_fpm,accepted requests
+php_fpm.processes.max_reached,count,,process,,Maximum number of processes reached,0,php_fpm,max processes reached
```

Comment:
Based on https://brandonwamboldt.ca/understanding-the-php-fpm-status-page-1603/ , I think a better description for this would be `the number of times the process limit has been reached, when pm tries to start more children`